content/en/docs/setup/install/multicluster/index.md

```diff
@@ -0,0 +1,724 @@
+---
+title: Multicluster Installation
+description: Install an Istio mesh across multiple Kubernetes clusters.
+weight: 30
+aliases:
+    - /docs/setup/kubernetes/multicluster-install/
+    - /docs/setup/kubernetes/multicluster/
+    - /docs/setup/kubernetes/install/multicluster/
+keywords: [kubernetes,multicluster]
+test: yes
+owner: istio/wg-environments-maintainers
+---
+When installing Istio across multiple clusters, there are more things to
+consider than with a [single cluster installation](/docs/setup/getting-started).
+
+Although there is no default installation procedure for a multicluster mesh, as it will
```

Comment:
I'd rephrase:

This guide covers some of the most common concerns when creating a multi-cluster mesh using Istio:
...

I don't think we need the first part about things being different.